scripts/create_testenv.sh

```diff
@@ -7,8 +7,8 @@ command -v conda >/dev/null 2>&1 || {
   exit 1;
 }
 
-# if no python specified, use Travis version, or else 3.8
-PYTHON_VERSION=${PYTHON_VERSION:-${TRAVIS_PYTHON_VERSION:-3.8}}
+# if no python specified, use Travis version, or else 3.9
+PYTHON_VERSION=${PYTHON_VERSION:-${TRAVIS_PYTHON_VERSION:-3.9}}
```

Comment:
We should probably rename from TRAVIS_PYTHON_VERSION to something else since were not using TravisCI anymore